### PR TITLE
Change content language & preliminary multilingual routing

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/components/LanguageSwitcher.js [hunk 1]
```diff
@@ -16,14 +17,18 @@ function LanguageSwitcher() {
     i18n.changeLanguage(target.value);
   };
 
+  useEffect(() => {
```
Comment (saifulislamdev) [inline src/components/LanguageSwitcher.js:20]: As a note, we will want to check for edge cases in the future. Especially, if this works in nested routes. No push needed.

File: src/components/header.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
```
Comment (saifulislamdev) [inline src/components/header.js]: I see you import this but don't use it. Is this for the future?

File: src/templates/landing-page.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
+
   return (
     <Suspense fallback="loading">
-      <I18nextProvider i18n={i18next}>
-        <Layout>
+      <I18nextProvider i18n={i18n}>
+        <Helmet
+          title={t("title")}
+          htmlAttributes={{ lang: i18n.language }}
+          defer={false}
```
Comment (saifulislamdev) [inline src/templates/landing-page.js]: What does the `defer` attribute do?

File: src/components/header.js (hunk 2 above)
Comment (aannyyaa) [reply src/components/header.js]: Yeah I was trying to use it in order to access i18n from the I18nextProvider. I need to look into it more!

File: src/templates/landing-page.js (hunk 3 above)
Comment (aannyyaa) [reply src/templates/landing-page.js]: Oh I actually don't need that - I'll delete it and the useContext hook. I realized defer is just suggested if you're developing using gatsby-plugin-offline

File: src/components/LanguageSwitcher.js (hunk 1 above)
Comment (aannyyaa) [reply src/components/LanguageSwitcher.js:20]: Yeah this is definitely preliminary just so I could see the language content change. We can look into best practices for multilingual routing in gatsby

**Pull request merged**